src/Groups/libraries/atlasgroups.jl

```diff
@@ -13,14 +13,28 @@ for `name` is available, and `MatrixGroup` otherwise.
 # Examples
 ```jldoctest
 julia> atlas_group("A5")  # alternating group A5
-Permutation group of degree 5 and order 60
+Permutation group of degree 5 and order 60 with 2 generators
+  (1,2)(3,4)
+  (1,3,5)
 
 julia> atlas_group(MatrixGroup, "A5")
 Matrix group of degree 4
   over prime field of characteristic 2
+with 2 generators
+  [1   0   0   0]
+  [0   0   1   0]
+  [0   1   0   0]
+  [1   1   1   1]
+
```

Comment:
Have you checked that things like `doctest_fix` work as expected if the printing of some object contains empty lines? In particular I would be interested in if this manages to fix typos in the second generator' matrix (aka after the empty line)